src/project/view/View.java

```diff
@@ -44,4 +76,33 @@ else if ((n instanceof Group || n instanceof PowerPlant) && !n.isConnected()){
         }
         return str;
     }
+    
+    public static void main(String[] args) throws FileNotFoundException {
+    	
+    	Network myNetwork = new Network(0,0,0);
+    	
+    	View myView = new View(myNetwork);
+    	
+    	Scanner sc = new Scanner(System.in);
```

Comment:
Rien à redire sinon que je vais ajouter  l'annotation `@SuppressWarnings("resource")` (normalement un flux doit être explicitement fermé afin d'éviter des fuites de ressources mais fermer System.in ne semble pas une bonne habitude même si en théorie ça ne devrait pas poser de problème dans notre cas).